src/item/abcmouse.md

```diff
@@ -0,0 +1,6 @@
+---
+title: 'ABCMouse'
+link: 'https://www.abcmouse.com/'
+summary: 'The Step-by-Step Learning Path presents the full ABCmouse.com curriculum in a carefully designed program of more than 850 lessons in ten levels. As your child completes each lesson, he or she is guided to the next one. Schools can request access for free [here](https://www.ageoflearning.com/schools.html) and all students can use the service.'
```

Comment:
Could we remove the markdown link here please? There’s no facility for them to be rendered yet :)

Reply:
ah of course! submitted!